pom.xml

```diff
@@ -28,6 +28,27 @@
 			<scope>runtime</scope>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
```

Comment:
If you use spring-boot-stater-web, you don't need spring-boot-started in POM. Please check what is inside spring-boot-starter-web

Reply:
Got it, I have removed spring-boot-starter